### DISCUSSION TITLE
GitHub Copilot support?

### DISCUSSION THREAD

**Discussion by georgeowenasurion** [coteditor/CotEditor#1548] [Q&A]
## GitHub Copilot support in CotEditor?

1. Does CotEditor support integration with GitHub Copilot at this time? 
(_It appears it does **NOT** at the moment_).
    > Current Supported Editors:
    > [Azure Data Studio](https://docs.github.com/en/copilot/getting-started-with-github-copilot?tool=azure_data_studio)
[JetBrains IDEs](https://docs.github.com/en/copilot/getting-started-with-github-copilot?tool=jetbrains)
[Vim/Neovim](https://docs.github.com/en/copilot/getting-started-with-github-copilot?tool=vimneovim)
[Visual Studio](https://docs.github.com/en/copilot/getting-started-with-github-copilot?tool=visualstudio)
[Visual Studio Code](https://docs.github.com/en/copilot/getting-started-with-github-copilot?tool=vscode)

3. Are there any plans to add official support for GitHub Copilot?

4. If there will be no official integration between Copilot and CotEditor, does anyone have a hack-y workaround to shoehorn it in unofficially? 
   > (Perhaps via some combination of scripts?)

Finally, thank you to all on the dev team who make CotEditor the best modern text-editor.

**Comment by 1024jp** [comment]
Hmmm, at the moment, I personally not interested in implementing GitHub Copilot...

**Comment by sihayas** [comment]
Any update on this?

**Reply by 1024jp** [reply]
No update.
